tests/languages/all_test.py

```diff
@@ -1,20 +1,34 @@
 from __future__ import unicode_literals
 
+import functools
 import inspect
+import sys
 
 import pytest
 
 from pre_commit.languages.all import all_languages
 from pre_commit.languages.all import languages
 
 
+if sys.version_info < (3,):
```

Comment:
will probably need `# pragma: no cover`s for this -- elsewhere `six.PY2` is being used, let's use that here as well